### ISSUE TITLE
Extra parameter sent when creating a Superset service

### ISSUE THREAD

**Issue by nahuelverdugo** [open-metadata/OpenMetadata#10398]
**Affected module**
UI

**Describe the bug**
When creating a Superset service with a Superset Connection which is Postgres or MySQL it fails to test the connection or to store it in DB. 

**To Reproduce**
After changing the Superset Connection to Postgres or MySQL, fill in the connection parameters and click test connection.

![image](https://user-images.githubusercontent.com/11074908/222472557-5082ebc2-aa0c-47ae-9902-7e0659b95f80.png)

**Expected behavior**
It should not fail.

**Version:**
 - OpenMetadata version: 0.13.2

**Additional context**
We are sending an additional parameter in the connection to be tested, in this case, it is `db`.

/cc @open-metadata/ui

**Comment by SandipDogra** [comment]
Any update

**Comment by SandipDogra** [comment]
```
Failed to test connection due to 
{"error": "Request Validation Error parsing payload. (Workflow)Source expected: 23 validation errors for SupersetConnection\nconnection 
-> classificationName\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> connectionArguments\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> connectionOptions\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> database\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> hostPort\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> scheme\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> supportsDBTExtraction\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> supportsDatabase\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> supportsLineageExtraction\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> supportsMetadataExtraction\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> supportsProfiler\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> supportsQueryComment\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> supportsUsageExtraction\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> type\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> provider\n extra fields not permitted (type=value_error.extra)\nconnection 
 -> type\n value is not a valid enumeration member;
  permitted: 'Mysql' (type=type_error.enum; enum_values=[<MySQLType.Mysql: 'Mysql'>])\nconnection 
  -> scheme\n value is not a valid enumeration member;
   permitted: 'mysql+pymysql' (type=type_error.enum; enum_values=[<MySQLScheme.mysql_pymysql: 'mysql+pymysql'>])\nconnection 
   -> classificationName\n extra fields not permitted (type=value_error.extra)\nconnection 
   -> database\n extra fields not permitted (type=value_error.extra)\nconnection 
   -> provider\n extra fields not permitted (type=value_error.extra)\nconnection 
   -> supportsDatabase\n extra fields not permitted (type=value_error.extra)\nconnection 
   -> supportsLineageExtraction\n extra fields not permitted (type=value_error.extra)\nconnection 
   -> supportsUsageExtraction\n extra fields not permitted (type=value_error.extra)"}
```

**Comment by harshach** [comment]
@Ashish8689 is this fixed?

**Comment by Ashish8689** [comment]
@harshach No, it's not . We have decided to fix this in 1.0.1 as we may also need to change the Ingestions scheme for  every services. 
cc: @chirag-madlani
